src/apis/DataInputApi.ts

```diff
@@ -0,0 +1,14 @@
+
+import axios from 'axios';
+import { DataInputRequest, DataInputResponse } from '../types/Types';
+
+const BASE_URL = 'https://api.example.com'; // Replace with your API base URL
```

Comment:
This is the URL that we should call: `https://75iaqg7l2j.execute-api.ap-south-1.amazonaws.com/prod`

Sample payload:
```
{
  "userId": "vap76",
  "firstName": "vaibhav",
  "lastName": "vasant",
  "email": "[v12@gmail.com](mailto:v12@gmail.com)",
  "phoneNumber": "7633824597",
  "address": "None"
}
```